### ISSUE TITLE
Field note visual height glitch on Chrome

### ISSUE THREAD

**Issue by fabriciomurta** [extnet/Ext.NET#1428]
Found: 4.2.0
Ext.NET forum thread: [Bug when displaying Field 3.x and 4.x ](http://forums.ext.net/showthread.php?61738)

On chrome, the spacing between fields looks broken, while it works fine in IE and FireFox.

This simple CSS override seems to address the issue:

```html
<style type="text/css">
    .x-form-item-body {
        height: initial
    }
</style>
```

**Comment by fabriciomurta** [comment]
Notice, this issue is not related to #1417 as it happens also on Ext.NET 3.x.

**Comment by fabriciomurta** [comment]
This use an specific Ext.NET feature, so there's no point in trying to reproduce in standalone ExtJS.

**Comment by fabriciomurta** [comment]
Removed the workaround in the example and the bug no longer happens. This was fixed in either ExtJS 6.5.0 or 6.5.1 releases.
